### DISCUSSION TITLE
What happens to all the resources?

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#1700] [General]
Seeing the forums are going to shut down: Where would all the current plugins go, and would there be another source for people to post/share their plugins like the TShock forums did?

**Comment by hakusaro** [comment]
Before we shut the forums down:

1. All of the plugins will be archived and the website will be preserved in some ways for access indefinitely.
2. We will have developed a new system to share plugins. We're probably going to go with a static site where devs can just send a PR to submit plugins, or just use GitHub tags. As an example, you can browse all of the repos tagged TShock, and developers can easily integrate with this by just publishing repos that are tagged a certain way: https://github.com/topics/tshock. This doesn't have the approval system but it's a good standin for what we have now.
